core/src/main/java/io/aiven/klaw/model/AclInfo.java

```diff
@@ -25,6 +25,7 @@ public class AclInfo {
   private String permission;
   private String transactionalId;
   private String aclPatternType;
+  private String aclId; // aiven acl id
   private String totalNoPages;
```

Comment:
if this is aiven specific should we specify it as aivenAclId ?

Reply:
Am trying to avoid 'aiven' specific variables. We might use the same when we integrate with confluent cloud.